genie-web/src/main/java/com/netflix/genie/web/apis/rest/v3/controllers/JobRestController.java

```diff
@@ -946,6 +946,8 @@ private String agentExecution(
 
         if (attachments != null) {
             jobSubmissionBuilder.withAttachments(
+                // Call AttachmentService.save here and stuff URIs into jobSubmission
```

Comment:
One of the main reasons I have for pushing this down into the `JobLaunchService` now is so that when the V4 API comes we don't have to either copy/paste all this into that controller or move it then and modify a bunch of existing code.